File: leetcode/100-999/171. Excel Sheet Column Number/solution_1.cpp
class Solution 
{
    public:
        int titleToNumber(string columnTitle) 
        {
            int title = 0;
            int power = 0;

            for (int i = columnTitle.size() - 1; i >= 0; --i)
            {
                int num = columnTitle[i] - 64;
                title += num * pow(26, power++);
            }   

            return title;
        }
};

File: leetcode/100-999/171. Excel Sheet Column Number/solution_1.js
/**
 * @param {string} columnTitle
 * @return {number}
 */

var titleToNumber = function(columnTitle) 
{
    let title = 0
    let power = 0

    for (const i = columnTitle.length - 1; i >= 0; --i)
    {
        const num = columnTitle[i] - 64
        title += num * Math.pow(26, power++)
    }    

    return title
};